packages/client/components/GitLabScopingSearchResults.tsx

```diff
@@ -120,89 +127,71 @@ const GitLabScopingSearchResults = (props: Props) => {
     query
   )
-  // const lastItem = useLoadNextOnScrollBottom(paginationRes, {}, 20)
-  const {data} = paginationRes
-  // const {viewer} = paginationRes
+
+  const lastItem = useLoadNextOnScrollBottom(paginationRes, {}, 20)
+  const {data, hasNext, loadNext, isLoadingNext} = paginationRes
   const {viewer} = data
-  // const meeting = useFragment(
-  //   graphql`
-  //     fragment GitLabScopingSearchResults_meeting on PokerMeeting {
-  //       # ...NewGitLabIssueInput_meeting
-  //       id
-  //       teamId
-  //       # gitlabSearchQuery {
-  //       #   queryString
-  //       # }
-  //       phases {
-  //         # ...useGetUsedServiceTaskIds_phase
-  //         phaseType
-  //       }
-  //     }
-  //   `,
-  //   meetingRef
-  // )
+  const meeting = useFragment(
+    graphql`
+      fragment GitLabScopingSearchResults_meeting on PokerMeeting {
+        # ...NewGitLabIssueInput_meeting
+        id
+        teamId
+        # gitlabSearchQuery {
+        #   queryString
+        # }
+        phases {
+          ...useGetUsedServiceTaskIds_phase
+          phaseType
+        }
+      }
+    `,
+    meetingRef
+  )
   const teamMember = viewer.teamMember!
   const {integrations} = teamMember
   const {gitlab} = integrations
-  // const {id: meetingId, gitlabSearchQuery, teamId, phases} = meeting
-  // const {phases} = meeting
+  const {id: meetingId, phases} = meeting
   // const {queryString} = gitlabSearchQuery
-  // const errors = gitlab?.api?.errors ?? null
+  const errors = gitlab?.api?.errors ?? null
   const nullableEdges = gitlab?.api?.query?.projects?.edges?.flatMap(
     (project) => project?.node?.issues?.edges ?? null
   )
   const issues = nullableEdges
     ? getNonNullEdges(nullableEdges)
         .filter((edge) => edge.node.__typename === '_xGitLabIssue')
-        .map(({node}) => node as GQLType<typeof node, '_xGitLabIssue'>)
+        .map(({node}) => node)
     : null
-  // const [isEditing, setIsEditing] = useState(false)
+  const [isEditing, setIsEditing] = useState(false)
   // const atmosphere = useAtmosphere()
-  // const estimatePhase = phases.find(({phaseType}) => phaseType === 'ESTIMATE')!
-  // const usedServiceTaskIds = useGetUsedServiceTaskIds(estimatePhase)
-  // const handleAddIssueClick = () => setIsEditing(true)
+  const estimatePhase = phases.find(({phaseType}) => phaseType === 'ESTIMATE')!
+  const usedServiceTaskIds = useGetUsedServiceTaskIds(estimatePhase)
+  const handleAddIssueClick = () => setIsEditing(true)
 
-  // even though it's a little herky jerky, we need to give the user feedback that a search is pending
-  // TODO fix flicker after viewer is present but edges isn't set
+  // when there are no issues in the initial query, gitlab can return an error with hasMore=false even though there are more projects
+  const noProjectsWithIssuesErrMsg = 'The user aborted a request.'
+  const errorMessage = errors?.[0]?.message
+  if (errorMessage === noProjectsWithIssuesErrMsg && !isLoadingNext) {
```

Comment:
When integrating with Parabol1, if there are no issues in the initial query, GitLab returns an error message. To test:

- in the pagination query, change the default value of `projectsFirst` to 1
- change the sort to `sort: "latest_activity_asc"`

You can play with the [explorer](https://gitlab.com/-/graphql-explorer) to experiment with this too.

I created a new test GitLab project, didn't add any issues, and ran the same query without getting an error message.

I don't like this solution as I'm not sure why the error happens with Parabol1, and the GitLab API could change the error message meaning `loadNext` wouldn't execute, so I'm open to other ideas. I've increased the default `projectsFirst` value from 5 to 10 to reduce the chance that this happens.